15-additional_resources.Rmd

```diff
@@ -0,0 +1,25 @@
+# Additional resources {#AdditionalResources}
+
+There are a number of further resources to help with contributions to R.
+
+## R Contributor site
+
+Contains information for people interested in contributing to R core, including details for the [R Contribution Working Group](https://contributor.r-project.org/working-group), the [R Contributors Slack workspace](https://contributor.r-project.org/slack) and upcoming [R Contributor events and meetups](https://contributor.r-project.org/events/). 
+
+## R developer page
+
+The [R developer page](https://developer.r-project.org/) contains details on the R release process, and the [R blog](https://blog.r-project.org/).
+
+## R developer mailing list
+
+You can find more information about, and subscribe to R development mailing list on the [R-devel site](https://stat.ethz.ch/mailman/listinfo/r-devel).
+
+## R Translations site
+
+There are a number of translation projects already underway or completed, more information on how to offer your help with one of these, contact others involved in translations, and to find more detailed instructions for translating R Messages or R Manuals can be found on the [R Translations site](https://contributor.r-project.org/translations/).
+
+## R manuals
+
+There are a number of manuals created and included with R installations. The original latest and development versions of these are available from the [R Manuals site](https://cran.r-project.org/manuals.html). Note that the versions included on the site are created with Debian Linux. While there may be some variation for Windows and Mac manuals,  most parts will be identical regardless of platform.
```

Comment:
Highlight the manuals that are of particular interest to R contributors and give some idea of what is of interest in each one.

Reply:
Added the 3 manuals you mentioned when we spoke earlier, let me know if the supporting text to them matches what you'd expect